map_gui/src/load.rs

```diff
@@ -452,6 +452,8 @@ where
     pub fn new(
```

Comment:
If we want to later, we can generalize this sort of two-level progress information. Maybe the current "inner" ones, used for downloading a single file, can just be a percentage and we draw a progress bar. But for now, this seems like a big enough step forward.